class BaseError(Exception):
        pass


class NotFound(BaseError):
        pass
